objects/tag.c

```diff
@@ -91,6 +91,83 @@
  *
  *    local t = awful.tag.find_by_name(awful.screen.focused(), "name")
  *
+ * **Common shortcuts**:
+ *
+ * Here is a few useful shortcuts not part of the default `rc.lua`. Add these
+ * functions above `-- {{{ Key bindings`:
+ *
+ * Delete the current tag
+ *
+ *    local function delete_tag()
+ *        local t = awful.screen.focused().selected_tag
+ *        if not t then return end
+ *        t:delete()
```

Comment:
Why not `if t then t:delete() end`? But whatever, I'll stop the bike-shedding here.